i18n/de.yaml

```diff
@@ -0,0 +1,5 @@
+- id: get-started
```

Comment:
Can we remove de.yaml and nl.yaml

Reply:
Should we remove the whole i18n until we need it?

Reply:
Yeah sounds good